docs/advanced/pool.rst

```diff
@@ -54,7 +54,7 @@ until the pool is full or will throw a `PoolTimeout` if the pool isn't ready
 within an allocated time.
 
 The pool background workers create connections according to the parameters
-*conninfo*, *kwargs*, *connection_class* passed to `ConnectionPool`
+*conninfo*, *kwargs* and *connection_class* passed to `ConnectionPool`
```

Comment:
I propose an Oxford comma here
```suggestion
*conninfo*, *kwargs*, and *connection_class* passed to `ConnectionPool`
```

Reply:
+1 on Oxford comma as a matter of personal opinion (though it's hard to be definitive on this unless the documentation is written to a particular style guide :-D)

Reply:
OK.